diff --git a/node_modules/widdershins/lib/openapi3.js b/node_modules/widdershins/lib/openapi3.js
index d673b05..74887c7 100644
--- a/node_modules/widdershins/lib/openapi3.js
+++ b/node_modules/widdershins/lib/openapi3.js
@@ -238,7 +238,7 @@ function getParameters(data) {
             requiredUriTemplateStr = requiredUriTemplateStr.split('{' + param.name + '}').join(template);
         }
         if (param.in === 'query') {
-            let isFirst = uriTemplateStr.indexOf('{&') < 0;
+            let isFirst = uriTemplateStr.indexOf('{?') < 0;
             // Since RFC6570 doesn't support multiple operators we cannot use (?+ and (&+ for reserved parameters
             let prefix = isFirst ? '{?' : '{&';
             var template = '';
@@ -248,7 +248,7 @@ function getParameters(data) {
             template += param.explode ? '*}' : '}';
             uriTemplateStr += (prefix + template);
 
-            if (param.required) {
+            if (param.required || param.example) {
                 let isFirstRequired = requiredUriTemplateStr.indexOf('{?') < 0;
                 let reqPrefix = isFirstRequired ? '{?' : '{&';
                 requiredUriTemplateStr += (reqPrefix + template);
